src/components/features/Button/Button.module.scss

```diff
@@ -0,0 +1,23 @@
+.btn {
+  align-items: center;
+  border-radius: 0.1rem;
+  border: none;
+  cursor: pointer;
+  display: flex;
+  font-size: 1rem;
+  font-weight: 600;
+  justify-content: center;
+  gap: 1rem;
+  line-height: 1.6 rem;
+  padding: 1rem 2rem;
+  white-space: nowrap;
+}
+
+.primary {
```

Comment:
Please take a look at this video and apply these classes as modifiers `.btn—primary`

https://www.youtube.com/watch?v=_2LwjfYc1x8